moj-node/server/views/pages/tags.html

```diff
@@ -58,4 +59,15 @@
     data: data.relatedContent,
     lazyLoadContent: true
   }) }}
+
+   <div class="govuk-form-group govuk-!-margin-bottom-2">
+    <p class="govuk-hub-article-feedback govuk-hub-feedback-text">Tell us what you think about this series:</p>
```

Comment:
Also wouldn't this appear for secondary tags?

You may want to check is the `contentType` is  series to display it